src/app/Instance/BridgeOverview/Components/EventProcessor.tsx

```diff
@@ -0,0 +1,40 @@
+import { Card, CardTitle, GridItem } from "@patternfly/react-core";
+import React from "react";
+import { OBEmptyState } from "./OBEmptyState";
+import { Data } from "../BridgeOverview";
+import { OBDashboardTableView } from "./OBDashboardTableView";
+
+interface EventProcessorProps {
+  EventProcessorList: Data[];
+  changeState: () => void;
```

Comment:
Instead of `changeState` I would choose a more meaningful name, like `onAddingProcessor` or something like that.